var/spack/repos/builtin/packages/aws-parallelcluster/package.py

```diff
@@ -11,8 +11,10 @@ class AwsParallelcluster(PythonPackage):
     tool to deploy and manage HPC clusters in the AWS cloud."""
 
     homepage = "https://github.com/aws/aws-parallelcluster"
-    url      = "https://pypi.io/packages/source/a/aws-parallelcluster/aws-parallelcluster-2.5.0.tar.gz"
+    url      = "https://pypi.io/packages/source/a/aws-parallelcluster/aws-parallelcluster-2.5.1.tar.gz"
+    maintainers = ['sean-smith', 'demartinofra', 'enrico-usai', 'lukeseawalker', 'rexcsn', 'ddeidda', 'tilne']
```

Comment:
This line is a bit too long:
```
var/spack/repos/builtin/packages/aws-parallelcluster/package.py:15: [E501] line too long (110 > 79 characters)
```
Can you wrap it onto a second line? We have special logic to ignore long URLs and checksums but not lists.

Reply:
@adamjstewart Sure no problem 👍